components/scream/src/share/tests/dummy_se_point_remapper.hpp

```diff
@@ -44,40 +44,46 @@ class DummySEPointRemapper : public AbstractRemapper<RealType>
 
   FieldLayout create_src_layout (const FieldLayout& tgt) const override {
     using namespace ShortFieldTagsNames;
-    auto ncol = this->get_src_grid()->get_native_dof_layout().dim(0);
+    using TV = std::vector<FieldTag>;
+    using IV = std::vector<int>;
+
+    auto ncol = this->get_src_grid()->get_2d_scalar_layout().dim(0);
 
     const auto rank = tgt.rank();
     EKAT_REQUIRE_MSG (rank==3 || rank==4 || rank==5,
       "Error! Target layout not supported. Remember that this class has limited support.\n");
     switch (rank) {
       case 3:
-        return FieldLayout ({COL},{ncol});
+        return FieldLayout (TV{COL},IV{ncol});
       case 4:
-        return FieldLayout ({COL,VL},{ncol,tgt.dim(3)});
+        return FieldLayout (TV{COL,VL},IV{ncol,tgt.dim(3)});
       case 5:
-        return FieldLayout ({COL,CMP,VL},{ncol,tgt.dim(1),tgt.dim(4)});
+        return FieldLayout (TV{COL,CMP,VL},IV{ncol,tgt.dim(1),tgt.dim(4)});
       default:
-        return FieldLayout ({},{});
+        return FieldLayout (TV{},IV{});
```

Comment:
Is this a meaningful field layout? Or does it indicate an error?

Reply:
The check at the top should throw an error before you hit this case. I think I added it to silence compiler warnings on CUDA maybe? I recall nvcc complaining for a fcn with no return if the default case was hit. I guess some compilers might not be smart enough to figure out that if you'd hit the default case, you'd have already errored out earlier on.